tests/lib/parse.js

```diff
@@ -23,10 +23,14 @@ describe("parse()", () => {
 
     describe("ecmaVersion", () => {
 
-        it("should be latestEcmaVersion if not specified", () => {
+        it("should be 5 if not specified", () => {
+            assert.throws(() => {
+                espree.parse(
+                    "let foo = bar;"
+                );
+
+            }, /Unexpected token foo/u);
```

Comment:
```suggestion
        it("should be 5 if not specified", () => {

            // needs `ecmaVersion: 5` or higher (`ecmaVersion: 3` would throw on getters/setters)
            espree.parse("var foo = { get bar() {} }");

            assert.throws(() => {
                espree.parse(

                    // needs `ecmaVersion: 6` or higher
                    "let foo = bar;"
                );

            }, /Unexpected token foo/u);
```

It would be nice to restore the check that it isn't < 5.